pkg/experiment/metastore/raftnode/raftnodepb/raft_node.proto

```diff
@@ -9,7 +9,14 @@ message RaftNode {
 
 service RaftNodeService {
   rpc ReadIndex(ReadIndexRequest) returns (ReadIndexResponse) {}
+}
+
+service RaftNodeOpsService {
   rpc NodeInfo(NodeInfoRequest) returns (NodeInfoResponse) {}
+  rpc RemoveNode(RemoveNodeRequest) returns (RemoveNodeResponse) {}
+  rpc AddNode(AddNodeRequest) returns (AddNodeResponse) {}
+  rpc DemoteLeader(DemoteLeaderRequest) returns (DemoteLeaderResponse) {}
+  rpc PromoteToLeader(PromoteToLeaderRequest) returns (PromoteToLeaderResponse) {}
 }
```

Comment:
I'm wondering if we need `RaftNodeOpsService`. I find it rather confusing.

Please consider extending the `RaftNodeService` service: all these methods provide access to the raft node. If you're concerned about the API surface, I can say that it's consumer responsibility to restrict it. So, for example, if you don't want the admin to have access to `ReadIndex` – it should define its own interface to the raft node client with only methods needed.

In the code I see that `RaftNodeOpsService` methods are already implemented on the `RaftNode` and `RaftNodeService` types. If you're 100% sure, a new separate service will make our solution better, please consider naming it `RaftAdminService` and refactoring its implementation out of the raft node package.

Reply:
done